index.html

```diff
@@ -12,28 +12,9 @@
         <div class="header__logo">
           <h1>Monster Finder</h1>
         </div>
-        <form class="search" id="search-monster-form">
-          <input
-            type="text"
-            class="search__field"
-            placeholder="Search your monster friends..."
-            autocomplete="off"
-            id="search-monster"
-          />
-          <button class="btn search__btn">
-            <svg class="search__icon">
-              <use href="/icons.svg#icon-magnifying-glass"></use>
-            </svg>
-            <span>Search</span>
-          </button>
-        </form>
-        <div></div>
```

Comment:
Likewise, a button isn't needed since we're looking at the 'input' event on the input field.